src/view/pages/Farmer/AddFarmer/index.tsx

```diff
@@ -129,12 +129,13 @@ class AddFarmer extends React.Component<PropsType, OwnStateType> {
    */
   private onChangePhoneNumber = (newPhone: string) => { 
     // let numbers = /^[0-9]+$/;
-    let numbers = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
-    if (!newPhone || !newPhone.match(numbers)) {
-      this.setState(state => ({ validPhoneNumber: false }));
-    } else {
-      this.setState(state => ({ phoneNumber: newPhone, validPhoneNumber: true }));
-    }
+    // TODO uncomment the following lines when phone number is required
+    // let numbers = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
+    // if (!newPhone || !newPhone.match(numbers)) {
+      // this.setState(state => ({ validPhoneNumber: false }));
+    // } else {
+    this.setState(state => ({ phoneNumber: newPhone, validPhoneNumber: true }));
+    // }
   }
```

Comment:
I think we should keep phone number as required. can you reduce the number of required characters down to 7?